tests/Integration/content-helper/ContentHelperEditorSidebarTest.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Integration Tests: PCH Editor Sidebar
+ *
+ * @package Parsely\Tests
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\ContentHelper;
+
+use Parsely\Content_Helper;
+
+/**
+ * Integration Tests for the PCH Editor Sidebar.
+ */
+final class ContentHelperEditorSidebarTest extends ContentHelperFeatureTest {
+	/**
+	 * Teardown method called after each test.
+	 */
+	public function tear_down(): void {
+		parent::tear_down();
+		wp_dequeue_script( Content_Helper::get_script_id() );
+		wp_dequeue_style( Content_Helper::get_style_id() );
```

Comment:
- This is not needed in `AdminColumnsParselyStatsTest` since some functions already address it.
- In `ContentHelperDashboardWidgetTest`, this is taken care of within the `assert_enqueued_status()` function [here](https://github.com/Parsely/wp-parsely/blob/f5db4851f6958aac2fccf044ba1d8c9aa654be0e/tests/Integration/content-helper/ContentHelperDashboardWidgetTest.php#L51-L55).

I'll remove the block you refer to and implement it into the `assert_enqueued_status()` function of `ContentHelperEditorSidebarTest` in order to follow a common approach with `ContentHelperDashboardWidgetTest`.